app/views/clients/index.html.slim

```diff
@@ -2,19 +2,17 @@ h1= t('clients')
 nav.navbar.section-navigation
   hr
   ul.list-inline
-    li= button_link t_title(:new), new_client_path, 'success', dimension: 'sm'
-    - if policy(Client).destroy?
-      li= button_link t('need_accompanying'), need_accompanying_clients_url, dimension: 'sm'
+    li= button_link 'Klient erfassen', new_client_path, dimension: 'sm'
+    = list_filter_dropdown(:salutation, Client.salutation_collection)
+    = list_filter_dropdown(:state, Client::STATES)
+    li= button_link t('clear_filters'), clients_path, dimension: 'sm'
+    li= button_link t('export_xlsx'), url_for(format: :xlsx, q: search_parameters), dimension: 'sm'
 
+    - if policy(Client).need_accompanying?
+      li= button_link 'Klienten ohne Begleitung', need_accompanying_clients_url, dimension: 'sm'
```

Comment:
Same here, or is this whole page only available in German?

Reply:
Yes